src/components/forms/form-elements/FormElements.vue

```diff
@@ -165,59 +165,59 @@
-                    </label>
-                  </div>
+                  <vuestic-checkbox
+                          :label="'forms.controls.subscribe' | translate"
+                          :id="'checkbox1'"
+                          v-model="checkboxOneModel">
+                  </vuestic-checkbox>
+                  <vuestic-checkbox
+                          :label="'forms.controls.subscribe' | translate"
+                          :id="'checkbox2'"
+                          v-model="checkboxTwoModel">
+                  </vuestic-checkbox>
+                  <vuestic-checkbox
+                          :label="'forms.controls.subscribe' | translate"
+                          :id="'checkbox3'"
+                          :disabled="true"
+                          v-model="checkboxThreeModel">
+                  </vuestic-checkbox>
+                  <vuestic-checkbox
+                          :label="'forms.controls.subscribe' | translate"
+                          :id="'checkbox4'"
+                          :disabled="true"
+                          v-model="checkboxFourModel">
+                  </vuestic-checkbox>
                 </fieldset>
               </div>
 
               <div class="col-md-3">
                 <fieldset>
-                  <div class="form-check abc-checkbox abc-checkbox-primary abc-checkbox-circle">
-                    <input class="form-check-input" id="checkbox7" type="checkbox">
-                    <label class="form-check-label" for="checkbox7">
-                      <span class="abc-label-text">{{'forms.controls.subscribe' | translate}}</span>
-                    </label>
-                  </div>
-                  <div class="form-check abc-checkbox abc-checkbox-primary abc-checkbox-circle">
-                    <input class="form-check-input" id="checkbox8" type="checkbox" checked>
-                    <label class="form-check-label" for="checkbox8">
-                      <span class="abc-label-text">{{'forms.controls.subscribe' | translate}}</span>
-                    </label>
-                  </div>
-                  <div class="form-check abc-checkbox abc-checkbox-primary abc-checkbox-circle">
-                    <input class="form-check-input" id="checkbox9" type="checkbox" disabled>
-                    <label class="form-check-label" for="checkbox9">
-                      <span class="abc-label-text">{{'forms.controls.subscribe' | translate}}</span>
-                    </label>
-                  </div>
-                  <div class="form-check abc-checkbox abc-checkbox-primary abc-checkbox-circle">
-                    <input class="form-check-input" id="checkbox10" type="checkbox" checked disabled>
-                    <label class="form-check-label" for="checkbox10">
-                      <span class="abc-label-text">{{'forms.controls.subscribe' | translate}}</span>
-                    </label>
-                  </div>
+                  <vuestic-checkbox
```

Comment:
Use self-closing tags for components without content https://vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended